MFVNeutralino/python/VertexSelector_cfi.py

```diff
@@ -98,7 +98,7 @@
     mevent_src = 'mfvEvent',
     min_ntracks = 3,
     #min_bsbs2ddist = 0.005,
-    max_rescale_bs2derr = 0.05,
+    max_rescale_bs2derr = 0.0025,
     )
```

Comment:
For what it's worth, I imagine the "ExtraLoose" vertices w/ a relaxed bs2derr could be more useful than just those with a relaxed dBV in most cases. But probably we won't be using these except in specific studies anyway, so it's fine as is and we can always adjust when needed.

Reply:
I used a tight bs2derr only because that is what I used for ML. It can be easily changed but I can change that to a relaxed value for the pull request.